nixos/modules/config/fonts/fontconfig.nix

```diff
@@ -183,6 +178,26 @@ let
     ln -s ${pkg.out}/etc/fonts/conf.d/*.conf \
           $dst/
 
+    ${optionalString (!cfg.antialias)
+      (replaceDefaultConfig "10-yes-antialias.conf" "10-no-antialias.conf")
+    }
+
+    ${optionalString (cfg.hinting.style != "slight")
+      (replaceDefaultConfig "10-hinting-slight.conf" "10-hinting-${cfg.hinting.style}.conf")
+    }
+
+    ${optionalString (cfg.subpixel.rgba != "none")
+      (replaceDefaultConfig "10-sub-pixel.conf" "10-sub-pixel-${cfg.subpixel.rgba}.conf")
+    }
+
+    ${optionalString (cfg.subpixel.lcdfilter != "default" && cfg.subpixel.lcdfilter != "none")
+      (replaceDefaultConfig "11-lcdfilter-default.conf" "11-lcdfilter-${cfg.subpixel.rgba}.conf")
+    }
+
+    ${optionalString (cfg.subpixel.lcdfilter == "none") ''
```

Comment:
Hmm, it’s weird that there is no `11-lcdfilter-none.conf`. I wonder if it is fine to not define it.

Reply:
I just pushed my commits again. This patch is now backported and the NixOS module doesn't have this special check anymore.